src/classes/CMT_MetadataAPI.cls

```diff
@@ -153,16 +174,14 @@ public class CMT_MetadataAPI {
                 isSuccess = false;
             }
 
-            // Write the deployment status to the custom settings object
-            Customizable_Rollup_Settings__c crlpSettings = UTIL_CustomSettingsFacade.getCustomizableRollupSettings();
-            crlpSettings.CMT_API_Status__c = JSON.serializePretty(statusMap).left(255);
-            // THIS WILL NEED TO BE REFACTORED IF THIS CMT DEPLOYMENT CODE IS USED BY SOMETHING OTHER THAN ROLLUPS
-            crlpSettings.Customizable_Rollups_Enabled__c = (isSuccess || crlpSettings.Customizable_Rollups_Enabled__c == true);
-            if (!Test.isRunningTest()) {
-                upsert crlpSettings;
-            }
             if (isSuccess) {
```

Comment:
This is the big change here - removed this CRLP specific code - as other calls of the CMT don't need / want this code running when deploying other Custom Metadata Types.